### ISSUE TITLE
When I use this plugin, I can't use Android device open my website

### ISSUE THREAD

**Issue by zhoujiate** [wp-statistics/wp-statistics#307]
If I enable this plugin, I get a **ERR_CONNECTION_RESET 500** report when I use Android device to open my website. Even when I use desktop browser in develop mode (to simulate Android device) I get the same problem. But when I use desktop browser in normal mode or iPhone, the website is working well. What's the problem?

When I disable this plugin, my website is fine.

My WP-Statistics version is 12.6.11 and my Wordpress verison is 5.3.
My PHP version is 7.3.9, my Apache version is 2.4.23 and my MySQL version is 5.7.14.

**Comment by mehrshaddarzi** [comment]
Hi @zhoujiate 
did you enable `use cache plugin` in WP-Statistics Option ?

**Comment by zhoujiate** [comment]
@mehrshaddarzi Hi,
I tried both enable and disable the "use cache plugin" and no use.
Before I try it, the "use cache plugin" is disabled.

**Comment by zhoujiate** [comment]
I got the problem file. Which is:
YOURWEBSITE\wp-content\plugins\wp-statistics\includes\vendor\whichbrowser\parser\data\models-android.php

When I delete all the content in "DeviceModels::$ANDROID_MODELS = []" the website can be browsered by android.

There must be something wrong around the []. Maybe the format mistake.

**Comment by zhoujiate** [comment]
OK, finally I found what's wrong.

It is because I opened the opcache to boost my PHP and the "models-android.php" file is too large(when this file smaller than 298kb the website will works fine).
I tried to adjust any variable in PHP.ini around opcache to make sure this file can be cached correctly but failed.

So there is my 2 solutions:
1. Shut down opcache in PHP.ini.
2. make a opcache blacklist and put "YOURWEBSITE\wp-content\plugins\wp-statistics\includes\vendor\whichbrowser\parser\data\models-android.php" in it.
